fluentui_drawer/src/main/java/com/microsoft/fluentui/bottomsheet/BottomSheet.kt

```diff
@@ -9,6 +9,7 @@ import android.app.Dialog
 import android.content.DialogInterface
 import android.os.Bundle
 import android.support.v7.app.AppCompatDialogFragment
+import java.util.*
```

Comment:
avoid using *

Reply:
This is copied as is from modular branch code. Corrected now